src/core/src/CoreMain.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, argv):
             composite_logger.log_debug("Building out full container...")
             container = bootstrapper.build_out_container()
             lifecycle_manager, telemetry_writer, status_handler = bootstrapper.build_core_components(container)
+            composite_logger.telemetry_writer = telemetry_writer  # Need to set telemetry_writer within composite logger to enable all logs to be sent to telemetry
             composite_logger.log_debug("Completed building out full container.\n\n")
```

Comment:
update comment: # Need to set telemetry_writer within logger to enable sending all logs to telemetry